solr/docker/docs/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+services:
+  solr:
+    image: solr
```

Comment:
Here and probably some README type places, the assumption is that the image is named "solr".  Yet it appears that you're deciding that the name "apache/solr" is the new default name.  I think pick one or or the other.

Reply:
I will pick apache/solr. It will be easier for us to find and replace if we decide to do something else later.

Reply:
I'm also going to go with the assumption that if we go with apache/solr, we will backfill all of the docker-solr releases to apache/solr. (So all of the previous version tags will be available under the new name.)